packages/web/src/app/doc-pages/brand/brand-routing.ts

```diff
@@ -15,6 +15,11 @@ export const routes: Routes = [
     path: Pages.Icon,
     loadComponent: () => import('./icon-doc/icon-doc.component').then((m) => m.IconDocComponent),
   },
+  {
+    path: Pages.Illustration,
+    loadComponent: () =>
+      import('./illustrations-doc/illustrations-doc.component').then((m) => m.IllustrationsDocComponent),
+  },
```

Comment:
When I first went to the preview I searched for the Illustration package in the side menu on `/components`.

Maybe we should add it to the list, but redirect the user to `/brand/Illustration`?